src/Compilers/CSharp/Test/Emit/PDB/PDBIteratorTests.cs

```diff
@@ -190,6 +198,7 @@ System.Collections.Generic.IEnumerable<int> IEI<T>(int i0, int i1)
           <namespace usingCount=""0"" />
         </using>
         <hoistedLocalScopes>
+          <slot />
           <slot startOffset=""0x2a"" endOffset=""0xb4"" />
           <slot startOffset=""0x6e"" endOffset=""0xb2"" />
```

Comment:
3 hoisted local scopes and no hoisted local fields?

Reply:
There were none before. I assume this is not because of the change?